Tests/BenchmarkTests/BenchmarkRunnerTests.swift

```diff
@@ -55,8 +55,7 @@ extension BenchmarkRunnerTests {
 
         var runner = BenchmarkRunner(
             suites: [suite1, suite2],
-            reporter: BlackHoleReporter(),
-            iterations: 1)
+            reporter: BlackHoleReporter())
```

Comment:
Please add a test for the new settings machinery! :-)

Reply:
Added in d2ccd08